lib/moon/design/loader.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Moon.Design.Loader do
+  @moduledoc false
+
+  use Moon.StatelessComponent
+
+  prop(size, :string, values: ["2xs", "xs", "md", "lg"], default: "md")
+  prop(color, :string, default: "hit", values: Moon.colors())
+
```

Comment:
please add class property and use it with merge function (last). see popover for example